readme.md

```diff
@@ -107,6 +107,30 @@ checkPassword('foo');
 //=> ArgumentError: Expected string `password` to have a minimum length of `6`, got `foo`
 ```
 
+### ow.validate(predicate)
```

Comment:
This needs a better name. Creating a "validator" is what `ow.create` does. So this creates confusion.